auth/server/src/main/java/com/seagates3/controller/IAMController.java

```diff
@@ -131,6 +132,98 @@ ServerResponse serve(FullHttpRequest httpRequest,
         LOGGER.error("Incorrect signature. Request not authenticated");
         return serverResponse;
       }
+    } else if (requestAction.equals("UpdateAccountLoginProfile")) {
+      LOGGER.debug("Parsing UpdateAccountLoginProfile request" +
+                   requestBody.get("AccountName"));
+      try {
+        clientRequestToken =
+            ClientRequestParser.parse(httpRequest, requestBody);
+      }
+      catch (InvalidAccessKeyException ex) {
+        LOGGER.debug(ex.getServerResponse().getResponseBody());
+        return ex.getServerResponse();
+      }
+      catch (InvalidArgumentException ex) {
+        LOGGER.debug(ex.getServerResponse().getResponseBody());
+        return ex.getServerResponse();
+      }
+
+      if (clientRequestToken == null) {
+        return responseGenerator.invalidArgument(
+            "AuthorizationHeaderMalformed");
+      }
+
+      try {
+        requestor = RequestorService.getRequestor(clientRequestToken);
+      }
+      catch (InvalidAccessKeyException ex) {
+        LOGGER.debug(ex.getServerResponse().getResponseBody());
+      }
+      catch (InternalServerException ex) {
+        LOGGER.debug(ex.getServerResponse().getResponseBody());
+      }
+      catch (InvalidRequestorException ex) {
+        LOGGER.debug(ex.getServerResponse().getResponseBody());
+      }
+      if (requestor == null) {
+        LOGGER.debug(
+            "Validating user on the basis of ldap credentials entered");
+        AccessKey akey = new AccessKey();
+        String ldapUser = AuthServerConfig.getLdapLoginCN();
+        if (!ldapUser.equals(clientRequestToken.getAccessKeyId())) {
+          LOGGER.error("Invalid ldap user, authentication failed");
```

Comment:
can we log the ldapUser value as well in the log

Reply:
we shoud not be logging credentials